io.openems.common/src/io/openems/common/jsonrpc/response/QueryHistoricTimeseriesExportXlsxResponse.java

```diff
@@ -32,131 +38,246 @@
+	// public static ChannelAddress ESS_SOC = new ChannelAddress("_sum", "EssSoc");
 
-	private static final int COL_WIDTH = 15;
+	public static Set<ChannelAddress> POWER_CHANNELS = Stream.of(//
+			GRID_ACTIVE_POWER, //
+			PRODUCTION_ACTIVE_POWER, //
+			CONSUMPTION_ACTIVE_POWER, //
+			ESS_ACTIVE_POWER).collect(Collectors.toCollection(HashSet::new));
 
-	private static byte[] generatePayload(ZonedDateTime fromDate, ZonedDateTime toDate,
-			SortedMap<ZonedDateTime, SortedMap<ChannelAddress, JsonElement>> historicData,
-			SortedMap<ChannelAddress, JsonElement> historicEnergy) throws IOException {
+	public static Set<ChannelAddress> ENERGY_CHANNELS = Stream.of(//
+			GRID_BUY_ENERGY, //
+			GRID_SELL_ENERGY, //
+			PRODUCTION_ACTIVE_ENERGY, //
+			CONSUMPTION_ACTIVE_ENERGY, //
+			ESS_ACTIVE_ENERGY // ,ESS_SOC
+	).collect(Collectors.toCollection(HashSet::new));
+
+	public static byte[] generatePayload(String edgeId, ZonedDateTime fromDate, ZonedDateTime toDate,
+			SortedMap<ZonedDateTime, SortedMap<ChannelAddress, JsonElement>> powerData,
+			SortedMap<ChannelAddress, JsonElement> energyData) throws IOException, OpenemsNamedException {
 		byte[] payload = new byte[0];
 		try (ByteArrayOutputStream os = new ByteArrayOutputStream()) {
 			Workbook workbook = new Workbook(os, "Historic data", "1.0");
 			Worksheet worksheet = workbook.newWorksheet("Sheet1");
 
-			initializeColWidths(worksheet, COL_WIDTH, historicData.size() + 1);
-
-			// TODO add Energy
-			addData(worksheet, 0, historicData);
+			addBasicInfo(worksheet, edgeId, fromDate, toDate);
+			addEnergyData(worksheet, energyData);
+			addPowerData(worksheet, powerData);
 
 			workbook.finish();
 			os.flush();
 			payload = os.toByteArray();
+
 			os.close();
 		}
 		return payload;
 	}
 
 	/**
-	 * Initializes the width of WorkSheet columns.
+	 * Adds basic information like the Edge-ID, date of creation,...
 	 * 
-	 * @param worksheet the WorkSheet
-	 * @param width     the width
-	 * @param columns   the number of columns
+	 * @param ws       the {@link Worksheet}
+	 * @param edgeId   the edgeId number
+	 * @param fromDate the fromdate the excel exported from
+	 * @param toDate   the todate the excel exported to
 	 */
-	public static void initializeColWidths(Worksheet worksheet, int width, int columns) {
-		for (int i = 0; i < columns; i++) {
-			worksheet.width(i, COL_WIDTH);
-		}
+	private static void addBasicInfo(Worksheet ws, String edgId, ZonedDateTime fromDate, ZonedDateTime toDate) {
+
+		DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/dd/yyyy");
+		String fromDateString = fromDate.format(formatter);
+		String toDateString = toDate.format(formatter);
+
+		addValueBold(ws, 0, 0, "FEMS-Nr.");
+		addValue(ws, 0, 1, edgId);
+		addValueBold(ws, 1, 0, "Export erstellt am");
+		addValue(ws, 1, 1, fromDateString);
+		addValueBold(ws, 2, 0, "Export Zeitraum");
+		addValue(ws, 2, 1, toDateString);
```

Comment:
"Export Zeitraum" means "period of export", i.e. derived from startDate and toDate